be/src/exec/vectorized/sorting/sort_column.cpp

```diff
@@ -289,6 +294,11 @@ class VerticalColumnSorter final : public ColumnVisitorAdapter<VerticalColumnSor
         return Status::OK();
     }
 
+    Status do_visit(const vectorized::StructColumn& column) {
+        // TODO(SmithCruise)
```

Comment:
ditto

Reply:
This class is using for column sort, and strcut column will not support sort operation.